site/source/docs/tools_reference/settings_reference.rst

```diff
@@ -3355,3 +3355,17 @@ Experimental support for wasm ESM integration.
 Requires EXPORT_ES6 and MODULARIZE=instance
 
 Default value: false
+
+.. _js_base64_api:
+
+JS_BASE64_API
+=============
+
+Enable to use of the proposed arraybuffer-base64 API:
```

Comment:
If it's already stage 3 at TC39, I think it's safe to drop the "proposed" qualifier here. IIUC, stage 3 is more or less equivalent to our phase 4, i.e. considered "done" and can be shipped to the web.